react-components/src/components/CacheProvider/idAndKeyTranslation.ts

```diff
@@ -28,6 +28,11 @@ export function createFdmKey(id: DmsUniqueIdentifier): FdmKey {
   return `${id.space}/${id.externalId}`;
 }
 
+export function fdmKeyToId(fdmKey: FdmKey): DmsUniqueIdentifier {
+  const [space, ...externalIdComponents] = split(fdmKey, '/');
```

Comment:
Would consider to add a comment mentioning that the API spec states that space can not contain `/`, but externalId can